Valeriia/Movies_database_project/src/components/Movies/Movies.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import MovieItem from "../MovieItem";
+import { IMovie } from "../../types";
+import "./Movies.scss";
+
+interface IMoviesProps {
+  movies: IMovie[];
+}
+
+const Movies = ({ movies }: IMoviesProps) => (
+  <div className="movies">
+    {movies.map((item) => {
+      return (
+        <Link to={"/film/" + item.id} key={item.id} className="movies__item">
+          <MovieItem {...item} />
```

Comment:
better to pass movie prop
```suggestion
          <MovieItem movie={item} />
``` or

```suggestion
          <MovieItem {...(item as IMovie)} />
```